collections-lists/FlatteningNestedCollectionInC#/FlatteningNestedCollectionInC#/Models/Certification.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace FlatteningNestedCollection.Models;
+public class Certification
```

Comment:
Add one line of space between namespace and record. Also since for all others we have used record type. For Certification also instead of class it can be record.

Reply:
Added a space line and changed Certification to record.